packages/gestalt/src/Masonry.js

```diff
@@ -22,20 +22,20 @@ type Props<T> = {|
    * grow to fill column space, and shrink to fit if below min columns.
    */
   columnWidth?: number,
+  /**
+   * The amount of vertical and horizontal space between each item, specified in pixels.
+   */
+  gutterWidth?: number,
   /**
    * A React component (or stateless functional component) that renders the item you would like displayed in the grid. This component is passed three props: the item's data, the item's index in the grid, and a flag indicating if Masonry is currently measuring the item. *Note that this [must be a stable reference!](https://www.developerway.com/posts/react-re-renders-guide#part3.1)* If using a component declared within a parent function component, you must use [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) to ensure a stable reference.
    */
-  comp: ComponentType<{|
+  Item: ComponentType<{|
```

Comment:
@dangerismycat Why are we capitalizing the prop name? should it be `item` rather than `Item`?

Reply:
@AlbertCarreras this is to indicate that it's not a regular JavaScript function, but a React component.

A variable that is a React component should be capitalized. For example:

```js
const Nav = type === 'top' ? TopNav : BottomNav;
return <Nav />
```

Reply:
Yeah, I knew that, wasn't sure about capitalized props

Reply:
@AlbertCarreras I added more context to the PR summary. Check out this Slack thread: https://pinterest.slack.com/archives/C13KLG5P0/p1661293861162029